modules/navier_stokes/src/variables/BernoulliPressureVariable.C

```diff
@@ -107,7 +122,7 @@ BernoulliPressureVariable::isDirichletBoundaryFace(const FaceInfo & fi,
     return true;
 
   if (isInternalFace(fi) && std::get<0>(NS::isPorosityJumpFace(*_eps, fi, time)))
-    // We choose to apply the Dirichlet condition for the upwind element
+    // We choose to apply the Dirichlet condition for the upwind
     return std::get<0>(elemIsUpwind(*elem, fi, time));
```

Comment:
Shouldn't we have logic here for the expansion / contraction?


Also you deleted the always "upwind" for jumps from the documentation but it s still just that here

Reply:
No this thing would not change which one is dirichlet and which one is extrapolated for the pressure. If I have a pressure outlet then the downwind is always extrapolated while the upwind is dirichlet every time. Only the magnitude of the pressure jump depends on if it is contraction/expansion throught the velocity taken.

Okay, I'll restore the relevant parts of the documentation!